consent/manager_sql.go

```diff
@@ -153,12 +154,13 @@ func (m *SQLManager) RevokeUserAuthenticationSession(ctx context.Context, user s
 }
 
 func (m *SQLManager) CreateForcedObfuscatedAuthenticationSession(ctx context.Context, s *ForcedObfuscatedAuthenticationSession) error {
-	tx, err := m.db.Beginx()
+	tx, err := m.db.BeginTxx(ctx, nil)
 	if err != nil {
 		return sqlcon.HandleError(err)
 	}
 
-	if _, err := m.db.Exec(
+	if _, err := tx.ExecContext(
```

Comment:
This was a bug I believe. I fixed it since I was knee deep in the code at this point already.

` m.db.Exec` is not going to be done in the TX created above.